src/components/Preferences/LocalIdentities.js

```diff
@@ -72,13 +73,17 @@ const SelectableLocalIdentities = React.memo(
       setAddressesSelected(initialAddressesSelected)
     }, [initialAddressesSelected])
 
+    const [opened, setOpened] = React.useState(false)
```

Comment:
I think we could declare this in `LocalIdentities` directly, since we are not using it in this component (apart from passing it to `LocalIdentities`)?

Also what would you think about renaming `opened` / `setOpened` to be more specific, maybe something like `modalOpened` or even `clearAllModalOpened`?